tap-gui/plugins/runtime-resource-visibility.hbs.md

```diff
@@ -12,8 +12,18 @@ Do one of the following actions to access the Runtime Resources Visibility plug-
 - [Install the Tanzu Application Platform Full or View profile](../../install-intro.md)
 - [Install Tanzu Application Platform without using a profile](../../install-intro.md) and then
   install [Tanzu Application Platform GUI separately](../install-tap-gui.md)
-- Disable realtime metrics for clusters without a metrics server by setting
-`skipMetricsLookup` to `true` for that cluster in `app.config.yaml`
+- Make sure to review the section [If you have a metrics server](#a-idmetrics-servera-if-you-have-a-metrics-server)
+
+## <a id="metrics-server"></a> If you have a metrics server
+
+By default, the Kubernetes API will be ignoring the metrics server on your cluster, if you want to access
```

Comment:
Suggesting some slight edits:

By default, the Kubernetes API will not attempt to use any metrics servers on your clusters. If you want to access
metrics information for a cluster, set `skipMetricsLookup` to `false` for that cluster in your `app-config.yaml`'s `kubernetes` section.

Reply:
I'll apply this change, thanks.